src/GeositeFramework/plugins/launchpad/style.css

```diff
@@ -26,3 +26,41 @@
     list-style-type: none;
     padding: 0;
 }
+
+.truncate-toggle {
+    padding: 2px !important;
+    font-size: 0.7em !important;
+    width: 35%;
+    float: none !important;
```

Comment:
I think `!important` should be reserved for extreme cases. Like when there is third party CSS we physically cannot modify. Using it everywhere makes styles harder to override (for example, in print mode).